### PR TITLE
Dockerfile: [breaking] changed tagging and versioning standard

### PR DESCRIPTION
**Pull request by bryanlatten**
In order to account for simultaneously managing multiple supported PHP major/minor combinations, additional tagging granularity is needed. 

New Formula:
PHP_MAJOR.PHP_MINOR-Major.Minor.Patch(-variant)

### CODE REVIEW

File: Dockerfile-72 [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    phpenmod overrides && \
+    # Enable NewRelic via Ubuntu symlinks, but disable in file. Cross-variant startup script uncomments with env vars.
+    # HACK: ensure file/folders exist so prep script still runs until NewRelic extension works with PHP 7.2
+    touch $CONF_PHPMODS/newrelic.ini && \
+    mkdir /var/log/newrelic && \
+    # phpenmod newrelic && \
+    # - Run standard set of tweaks to ensure runs performant, reliably, and consistent between variants
+    /bin/bash -e /prep-php.sh
+
+RUN goss -g /tests/php-fpm/72.goss.yaml validate && \
```
Comment (bossjones) [inline Dockerfile-72]: Missing period: `goss -g /tests/php-fpm/7.2.goss.yaml validate`

Comment (bossjones) [reply Dockerfile-72]: also missing variant? Since this is ubuntu ?

File: Dockerfile-71-alpine [hunk 2]
```diff
@@ -148,5 +148,5 @@ RUN apk update && \
     apk del .phpize_deps && \
     /bin/bash -e /clean.sh
 
-RUN goss -g /tests/php-fpm/alpine.goss.yaml validate && \
+RUN goss -g /tests/php-fpm/71-alpine.goss.yaml validate && \
```
Comment (bossjones) [inline Dockerfile-71-alpine]: Missing period: `goss -g /tests/php-fpm/7.1.goss.yaml validate`

File: Dockerfile-70 [hunk 3]
```diff
@@ -124,6 +124,6 @@ RUN ln -s /usr/sbin/php-fpm7.0 /usr/sbin/php-fpm && \
     # Run standard set of tweaks to ensure runs performant, reliably, and consistent between variants
     /bin/bash -e /prep-php.sh
 
-RUN goss -g /tests/php-fpm/ubuntu.goss.yaml validate && \
+RUN goss -g /tests/php-fpm/70.goss.yaml validate && \
```
Comment (bossjones) [inline Dockerfile-70]: Missing period: `goss -g /tests/php-fpm/7.0.goss.yaml validate`

File: Dockerfile-56 [hunk 4]
```diff
@@ -125,5 +125,5 @@ RUN cp /etc/php/7.0/mods-available/* $CONF_PHPMODS && \
     # - Run standard set of tweaks to ensure runs performant, reliably, and consistent between variants \
     /bin/bash -e /prep-php.sh
 
-RUN goss -g /tests/php-fpm/legacy.goss.yaml validate && \
+RUN goss -g /tests/php-fpm/56.goss.yaml validate && \
```
Comment (bossjones) [inline Dockerfile-56]: Missing period: `goss -g /tests/php-fpm/5.6.goss.yaml validate`

**Comment by jimdelois** [comment]
👍

File: Dockerfile-56 (hunk 4 above)
Comment (bryanlatten) [reply Dockerfile-56]: hm...I wonder what's wrong in git. That's been updated (for all of these) in my branch

**Comment by bossjones** [comment]
👍

**Comment by mannytoledo** [comment]
👍

File: Dockerfile-72 [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+        patch \
+        software-properties-common \
+        locales \
+        wget \
+    && \
+    /bin/bash /clean.sh
+
+# Add PHP and support packages \
+RUN apt-get update -q && \
+    # Ensure PHP 5.5 + 5.6 + 7.1 don't accidentally get added by PPA
```
Comment (ko-be) [inline Dockerfile-72:56]: I think you mean `PHP 5.6 + 7.0 + 7.1` here

**Pull request merged**